### PR TITLE
Add park and movinghome modules to MAVProxy docs

### PR DESCRIPTION
**Pull request by Hwurzburg**

### CODE REVIEW

File: mavproxy/source/docs/modules/movinghome.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+.. code:: bash
+
+    module load movinghome
+
+Usage
+=====
+
+movinghome <status|on|off|radius|device|baud>
+
+- on: enable position updates as GCS moves. If GSC moves beyond thresold, vehicle home will be updated to current GCS position.
```
Comment (AndKe) [inline mavproxy/source/docs/modules/movinghome.rst:18]: typo in "threshold" (I know, any typos are most likely copied from my document)

**Review by AndKe — COMMENTED** [review]
Looks good,  I commented on one typo.

File: mavproxy/source/docs/modules/movinghome.rst (hunk 1 above)
Comment (Hwurzburg) [reply mavproxy/source/docs/modules/movinghome.rst]: fixed

**Comment by AndKe** [comment]
@Hwurzburg  Thank you - have a great weekend.

**Pull request merged**